lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -437,6 +441,8 @@ abstract class PrimaryRoleManager {
   /// management intereferes with the widget's functionality.
   PrimaryRoleManager.blank(this.role, this.semanticsObject);
 
+  late final DomElement element = _createAndInitElement();
```

Comment:
non-late final can only accept static function as its initialize. In this case, there isn't a way for sub class to inject their logic

Reply:
Oh I see, you need to call `this.createElement` which not static. In that case `late` make sense. However, I'd still initialize it in the constructor. Otherwise, it gets initialized upon first read of the field, and since it's hard to predict when exactly that field is accessed we can get into race conditions, such as `_createAndInitElement` reading from `semanticsObject` that has changed its state after the constructor is finished. Can you move `element = _createAndInitElement();` into the constructor body?